src/app/onion/learning-object-builder/components/editor-action-panel/change-status-modal/change-status-modal.component.ts

```diff
@@ -77,13 +77,15 @@ export class ChangeStatusModalComponent implements OnInit {
         ];
         break;
       case LearningObject.Status.ACCEPTED_MINOR:
-        this.statuses = [LearningObject.Status.WAITING];
-        break;
       case LearningObject.Status.ACCEPTED_MAJOR:
-        this.statuses = [LearningObject.Status.WAITING];
+        this.statuses = [LearningObject.Status.WAITING, LearningObject.Status.PROOFING];
         break;
       case LearningObject.Status.PROOFING:
         this.statuses = [LearningObject.Status.RELEASED, LearningObject.Status.REJECTED];
+        break;
+      case LearningObject.Status.RELEASED:
+        this.statuses = [LearningObject.Status.PROOFING];
+        break;
```

Comment:
This is actually handled by a separate function in the admin dashboard so it is not needed here.